### DISCUSSION TITLE
Milvus connection error in windows

### DISCUSSION THREAD

**Discussion by jaylmilan** [milvus-io/milvus#15964] [Q&A and General discussion]
![image](https://user-images.githubusercontent.com/84993840/157558697-9a3b0316-04f4-4f44-af85-6cc763acbb4e.png)

Followed the instruction in https://milvus.io/blog/2021-11-19-run-milvus-2.0-on-windows.md#Install-from-the-binary-package

Getting this error ,
PS C:\Users\jaylm\Downloads\milvus-windows-20211123-15f264f\milvus> C:\Users\jaylm\anaconda3\envs\thirdeye\python.exe .\hello_milvus.py.py

=== start connecting to Milvus     ===

RPC error: [has_collection], <BaseException: (code=1, message=proxy not healthy)>, <Time:{'RPC start': '2022-03-09 18:15:09.345488', 'RPC error': '2022-03-09 18:15:09.350488'}>
Traceback (most recent call last):
  File "C:\Users\jaylm\Downloads\milvus-windows-20211123-15f264f\milvus\hello_milvus.py.py", line 34, in <module>
    has = utility.has_collection("hello_milvus")
  File "C:\Users\jaylm\anaconda3\envs\thirdeye\lib\site-packages\pymilvus\orm\utility.py", line 351, in has_collection
    return _get_connection(using).has_collection(collection_name)
  File "C:\Users\jaylm\anaconda3\envs\thirdeye\lib\site-packages\pymilvus\decorators.py", line 56, in handler
    raise e
  File "C:\Users\jaylm\anaconda3\envs\thirdeye\lib\site-packages\pymilvus\decorators.py", line 41, in handler
    return func(self, *args, **kwargs)
  File "C:\Users\jaylm\anaconda3\envs\thirdeye\lib\site-packages\pymilvus\decorators.py", line 74, in handler
    raise e
  File "C:\Users\jaylm\anaconda3\envs\thirdeye\lib\site-packages\pymilvus\decorators.py", line 70, in handler
    return func(*args, **kwargs)
  File "C:\Users\jaylm\anaconda3\envs\thirdeye\lib\site-packages\pymilvus\client\grpc_handler.py", line 169, in has_collection
    raise BaseException(reply.status.error_code, reply.status.reason)
pymilvus.client.exceptions.BaseException: <BaseException: (code=1, message=proxy not healthy)>
PS C:\Users\jaylm\Downloads\milvus-windows-20211123-15f264f\milvus>

Any suggestion to resolve them. Thanks.

**Comment by yhmo** [comment]
The error "proxy not healthy" indicates the service is not ready, or dependent services(etcd, minio) are not available. Please check the state of the etcd service and minio service. If the two services are working fine, try run the example again.

**Reply by jaylmilan** [reply]
Logs and screen shot from etcd, minio and milvus exe
![image](https://user-images.githubusercontent.com/84993840/157584976-c6ff706b-4b3c-4494-85de-8c5c0a9f4db9.png)
![image](https://user-images.githubusercontent.com/84993840/157584997-4c9a9233-0731-451d-8ac4-329d0f65a4ce.png)
[milvus_log.log](https://github.com/milvus-io/milvus/files/8220088/milvus_log.log)

**Reply by jaylmilan** [reply]
I picked it from here
https://github.com/matrixji/milvus/releases

**Reply by matrixji** [reply]
Maybe I need to tag a new release for recent adaptations. as that milvus's version is too old, but I guess you're using the latest pymilvus, right?

**Reply by jaylmilan** [reply]
Yes. I am using Pymilvus 2.0.1. Thanks

**Comment by matrixji** [comment]
Please have a try with https://github.com/matrixji/milvus/releases/tag/windows-test-v27 this windows binary should be worked with the hello example of pymilvus==2.0.1
